xmr-btc/tests/harness/storage.rs

```diff
@@ -0,0 +1,158 @@
+use anyhow::{anyhow, Context, Result};
+use serde::{de::DeserializeOwned, Deserialize, Serialize};
+use std::path::Path;
+
+pub struct Database<T>
```

Comment:
Is this on purpose in `tests/harness/`? The db should be used in the binary later-on right?

Reply:
yeh, the test hardness was the closest thing we had to a binary when i submitted this